cmd/derper/derper.go

```diff
@@ -58,7 +58,11 @@ func loadConfig() config {
 		return config{PrivateKey: mustNewKey()}
 	}
 	if *configPath == "" {
-		log.Fatalf("derper: -c <config path> not specified")
+		if os.Geteuid() == 0 {
```

Comment:
does the `e` part matter? I think we usually just test `Getuid`.

Reply:
not in particular, just saw it on SO so used the e